src/frontends/onnx/frontend/src/utils/common.cpp

```diff
@@ -10,6 +10,19 @@
 #include "ngraph/graph_util.hpp"
 #include "onnx_framework_node.hpp"
 #include "openvino/core/deprecated.hpp"
+#include "openvino/frontend/exception.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/broadcast.hpp"
+#include "openvino/op/concat.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/logical_and.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/range.hpp"
+#include "openvino/op/reshape.hpp"
+#include "openvino/op/shape_of.hpp"
+#include "openvino/op/subtract.hpp"
```

Comment:
Why ops headers are required? Looks like `default_opset` that includes opset12 should be enough

Reply:
Looks like it is a rebasing artifact. It is part of further pr: removing default_opset.hpp for compilation optimization.
I’ll remove it here and fix code style.